packages/modules/order/src/services/order-module-service.ts

```diff
@@ -2807,4 +2610,76 @@ export default class OrderModuleService<
 
     return Array.isArray(orderId) ? orders : orders[0]
   }
+
+  // ------------------- Bundled Order Actions
```

Comment:
question: Could I get you to explain in a few sentences what we changed in this file?

Reply:
I just reorganized the functions to be separated by files in another folder. It was really hard to read.
The behavior is the same.